numba/tests/test_unicode.py

```diff
@@ -933,6 +937,17 @@ def pyfunc(x):
             self.assertEqual(pyfunc(*args), cfunc(*args),
                              msg='failed on {}'.format(args))
 
+    def test_title(self):
+        pyfunc = title
+        cfunc = njit(pyfunc)
+        # Samples taken from CPython testing:
+        # https://github.com/python/cpython/blob/3.6/Lib/test/test_unicode.py#L813-L828
+        cpython = ['\U0001044F', '\U0001044F\U0001044F', '\U0001044F\U0001044F \U0001044F\U0001044F',
+                   '\U00010427\U0001044F \U00010427\U0001044F', '\U0001044F\U00010427 \U0001044F\U00010427',
+                   'X\U00010427x\U0001044F X\U00010427x\U0001044F', 'ﬁNNISH', 'A\u03a3 \u1fa1xy', 'A\u03a3A']
+        for a in UNICODE_EXAMPLES + [''] + cpython:
```

Comment:
Suggest to add string with spaces and string with special characters.

Reply:
List `UNICODE_EXAMPLES` contains string with spaces and special characters.